.travis.yml

```diff
@@ -96,7 +96,7 @@ jobs:
     - cd ../..
     - cd backend/sprint
     - pylint **/*.py --load-plugins pylint_django
-    - coverage run --source='.' manage.py test
+    - coverage run --source='.' manage.py test --keepdb
```

Comment:
Is it safe to keep the DB? How does `--keepdb` command exactly affect the test?

Reply:
Django always creates test database to test. However in mysql, to create database needs a special privilege. So I made test database in advance and grant it to django. That's why I add keepdb option.

Reply:
Meaning that the test database will not be dropped, but just clear all the data after the end of the test?

Reply:
Exactly!